src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -747,6 +747,18 @@ export default {
 		flex-wrap: wrap;
 		box-sizing: border-box;
 		width: 100%;
+		
+		&.app-navigation-entry__settings-quota {
```

Comment:
Shouldnt this rather go into thr server repo? Also cc @skjnldsv

Reply:
Hmpf, that being said it's broken already in 26